spec.html

```diff
@@ -561,16 +559,12 @@ <h1><ins>GetCycleRoot ( _module_ )</ins></h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-cyclicmoduleexecutionfulfilled" aoid="CyclicModuleExecutionFulfilled">
-      <h1><ins>CyclicModuleExecutionFulfilled ( _module_ )</ins></h1>
+    <emu-clause id="sec-asyncmodulexecutionfulfilled" aoid="AsyncModuleExecutionFulfilled">
+      <h1><ins>AsyncModuleExecutionFulfilled ( _module_ )</ins></h1>
       <emu-alg>
-        1. Assert: _module_.[[Status]] is `"evaluating"`, `"evaluating-async"`, or `"evaluated"`.
+        1. Assert: _module_.[[Status]] is `"evaluated"`.
         1. Assert: _module_.[[EvaluationError]] is *undefined*.
-        1. If _module_.[[Status]] is `"evaluating"`, then
-          1. Assert: _module_.[[Async]] is *false*.
-          1. Assert: _module_.[[AsyncParentModules]] is an empty List.
-        1. If _module_.[[Status]] is `"evaluating-async"`, then
-          1. Set _module_.[[Status]] to `"evaluated"`.
+        1. Set _module_.[[Status]] to `"evaluated"`.
```

Comment:
If the assertion above is correct, this is a no-op. The status can still be "evaluating-async" at this point, though.

(I know this is all changing again in #114, but let's try to keep this PR purely editorial.)